indico/modules/receipts/util.py

```diff
@@ -233,7 +233,7 @@ def _get_default_value(field):
     return field['attributes'].get('value', '')
 
 
-def get_dummy_preview_data(custom_fields: dict) -> dict:
+def get_dummy_preview_data(custom_fields: dict, custom_fields_values: dict = {}) -> dict:
```

Comment:
just in case you didn't see the linter warning ;)

Reply:
Hmm I might have something wrong with my linter setup...